sql-ballerina/Package.md

```diff
@@ -1,371 +1,16 @@
-the client.
-
-```ballerina
-string name = "John";
-sql:ParameterizedQuery query = `DELETE from students WHERE name = ${name}`;
-var ret = dbClient->execute(query);
-if (ret is sql:ExecutionResult) {
-    io:println("Deleted student count: ", ret.affectedRowCount);
-} else {
-    error err = ret;
-    io:println("Delete from students table failed: ", err.message());
-}
-```
-
-#### Batch updating data
-
-This example demonstrates how to insert multiple records with a single INSERT statement that is executed via the 
-`batchExecute` remote function of the client. This is done by creating a `table` with multiple records and 
-parameterized SQL query as same as the  above `execute` operations.
-
-```ballerina
-// Create the table with the records that need to be inserted.
-var data = [
-  { name: "John", age: 25  },
-  { name: "Peter", age: 24 },
-  { name: "jane", age: 22 }
-];
-
-// Do the batch update by passing the batches.
-sql:ParameterizedQuery[] batch = from var row in data
-                                 select `INSERT INTO students ('name', 'age')
-                                 VALUES (${row.name}, ${row.age})`;
-var ret = dbClient->batchExecute(batch);
-
-if (ret is error) {
-    io:println("Error occurred:", err.message());
-} else {
-    io:println("Batch item 1 update count: ", ret[0].affectedRowCount);
-    io:println("Batch item 2 update count: ", ret[1].affectedRowCount);
-}
-```
-
-#### Execute SQL stored procedures
-
-This example demonstrates how to execute a stored procedure with a single INSERT statement that is executed via the 
-`call` remote function of the client.
-
-```ballerina
-int uid = 10;
-sql:IntegerOutParameter insertId = new;
-
-var ret = dbClient->call(`call InsertPerson(${uid}, ${insertId})`);
-if (ret is error) {
-    io:println("Error occurred:", err.message());
-} else {
-    io:println("Outparameter insert id: ", insertId.get(int));
-    stream<record{}, sql:Error>? resultStr = ret.queryResult;
-    if (!(resultStr is ())) {
-        sql:Error? e = resultStr.forEach(function(record{} result) {
-        io:println("Full Customer details: ", result);
-      });
-    } else {
-        io:println("Stored  procedure does not return anything.");
-    }
-    check ret.close();
-}
-```
-Note that you have to explicitly invoke the close operation on the `sql:ProcedureCallResult` to release the connection resources and avoid a connection leak as shown above.
-
->**Note:** The default thread pool size used in Ballerina is the number of processors available * 2. You can configure
-the thread pool size by using the `BALLERINA_MAX_POOL_SIZE` environment variable.
+### Useful Links
+- Discuss the code changes of the Ballerina project in ballerina-dev@googlegroups.com.
+- Chat live with us via our [Slack channel](https://ballerina.io/community/slack/).
+- Post all technical questions on Stack Overflow with the #ballerina tag.
```

Comment:
```suggestion
- Post all technical questions on Stack Overflow with the [#ballerina](https://stackoverflow.com/questions/tagged/ballerina) tag.
```

Reply:
resolved in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/0f8470ef0b323062b1f4e7a97722e4cc085e0d07